pontoon/api/schema.py

```diff
@@ -137,26 +142,27 @@ class Query(graphene.ObjectType):
     locale = graphene.Field(Locale, code=graphene.String())
 
     def resolve_projects(obj, info, include_disabled, include_system):
-        qs = ProjectModel.objects
         fields = get_fields(info)
 
+        projects = ProjectModel.objects.visible_for(info.context.user)
+        records = projects.filter(disabled=False, system_project=False)
+
+        if include_disabled:
+            records |= projects.filter(disabled=include_disabled)
```

Comment:
I'd be consistent with `resolve_localizations()` here and below (`if include_system`) and either hardcode `True` or use `include_disabled` / `include_system` also in `resolve_localizations()`.